java/mtsj/core/src/test/java/com/devonfw/application/mtsj/dishmanagement/logic/impl/DishmanagementTest.java

```diff
@@ -26,41 +30,57 @@
 @SpringBootTest(classes = SpringBootApp.class)
 public class DishmanagementTest extends ComponentTest {
 
-  @Inject
-  private Dishmanagement dishmanagement;
+	@Inject
+	private Dishmanagement dishmanagement;
 
-  /**
-   * This test gets all the available dishes using an empty SearchCriteria object
-   */
-  @Test
-  public void findAllDishes() {
+	/**
+	 * Login to get access to Dishmanagement methods called in Test Methods
+	 */
+	@BeforeClass
+	public static void login() {
+		TestUtil.login("user", ApplicationAccessControlConfig.PERMISSION_FIND_DISH);
```

Comment:
the preferred way in devonfw is to override `doSetUp` and `doTearDown` instead:
https://github.com/devonfw/devon4j/blob/develop/modules/test/src/main/java/com/devonfw/module/test/common/base/BaseTest.java#L13

Reply:
```
@Override
protected void doTearDown() {
  TestUtil.logout();
}
```
Would make perfect sense.
We could also create our own `ApplicationComponentTest` class to centralize this and derive all component tests from that instead.